### ISSUE TITLE
For maintainability, an extension to the submodule that just contains an activation function and dropout and batch norm should be made.

### ISSUE THREAD

**Issue by erikhuck** [byu-dml/d3m-dynamic-neural-architecture#73]

**Comment by bjschoenfeld** [comment]
You just want to group the activation, dropout, and batch norm in a single object?

**Comment by erikhuck** [comment]
Exactly. It avoids code duplication and this way we don't have to worry about them being used in the correct order. We also can customize whether dropout, batchnorm, or the activation are used or any combination of them in one object.

**Comment by erikhuck** [comment]
After reviewing brandon's refactor of the hidden_dag_lstm, I don't see this issue as necessary anymore
